src/listener/ClusterViewListenerService.ts

```diff
@@ -37,20 +38,23 @@ export class ClusterViewListenerService {
     private readonly partitionService: PartitionServiceImpl;
     private readonly logger: ILogger;
     private readonly invocationService: InvocationService;
+    protected readonly connectionRegistry: ConnectionRegistry;
```

Comment:
Why is this field marked as `protected`?

Reply:
by mistake